dicomImport/matRad_importDicomCt.m

```diff
@@ -178,24 +182,33 @@
 
 %% interpolate cube
 fprintf('\nInterpolating CT cube...');
-ct = matRad_interpCtCube(origCt, info, resolution);
+ct = matRad_interpCtCube(origCt, ctInfo, resolution);
 fprintf('finished!\n');
 
 %% remember some parameters of original dicom
-ct.dicomInfo.PixelSpacing            = info(1).PixelSpacing;
-                                       tmp = [info.ImagePositionPatient];
+ct.dicomInfo.PixelSpacing            = ctInfo(1).PixelSpacing;
+                                       tmp = [ctInfo.ImagePositionPatient];
 ct.dicomInfo.SlicePositions          = tmp(3,:);
-ct.dicomInfo.SliceThickness          = [info.SliceThickness];
-ct.dicomInfo.ImagePositionPatient    = info(1).ImagePositionPatient;
-ct.dicomInfo.ImageOrientationPatient = info(1).ImageOrientationPatient;
-ct.dicomInfo.PatientPosition         = info(1).PatientPosition;
-ct.dicomInfo.Width                   = info(1).Width;
-ct.dicomInfo.Height                  = info(1).Height;
-ct.dicomInfo.RescaleSlope            = info(1).RescaleSlope;
-ct.dicomInfo.RescaleIntercept        = info(1).RescaleIntercept;
+ct.dicomInfo.SliceThickness          = [ctInfo.SliceThickness];
+ct.dicomInfo.ImagePositionPatient    = ctInfo(1).ImagePositionPatient;
+ct.dicomInfo.ImageOrientationPatient = ctInfo(1).ImageOrientationPatient;
+ct.dicomInfo.PatientPosition         = ctInfo(1).PatientPosition;
+ct.dicomInfo.Width                   = ctInfo(1).Width;
+ct.dicomInfo.Height                  = ctInfo(1).Height;
+ct.dicomInfo.RescaleSlope            = ctInfo(1).RescaleSlope;
+ct.dicomInfo.RescaleIntercept        = ctInfo(1).RescaleIntercept;
+if isfield(completeDicom,'PatientName')
+    ct.dicomInfo.PatientName         = completeDicom.PatientName;
+end
 
+
+ct.dicomInfo_org = completeDicom;
+ct.timeStamp = datetime;
```

Comment:
I've used 'clock' at the beginning until I recognized the 'datetime' function and unfortunately missed to used it consistently. I think datetime uses a more 'natural' format for displaying the time and date.